pyannote/audio/features/spec_augmentor.py

```diff
@@ -0,0 +1,96 @@
+import numpy as np
```

Comment:
I have just created a pull request that adds support for user-defined callbacks:
https://github.com/pyannote/pyannote-audio/pull/195

Let's discuss tomorrow how to turn SpecAugmentor into a callback.